climakitae/core/data_interface.py

```diff
@@ -946,122 +1035,148 @@ def _update_unit_options(self):
-        scenario_historical_options = [
-            scenario_to_experiment_id(scen, reverse=True)
-            for scen in self.scenario_options
-            if scen in historical_scenarios
-        ]
-        self.param["scenario_historical"].objects = scenario_historical_options
+            # Set incoming scenario_historical
+            _scenario_historical = self.scenario_historical
+
+            # Get scenario options in catalog format
+            scenario_ssp_options = [
+                scenario_to_experiment_id(scen, reverse=True)
+                for scen in self.scenario_options
+                if "ssp" in scen
+            ]
+            for scenario_i in [
+                "SSP 3-7.0 -- Business as Usual",
+                "SSP 2-4.5 -- Middle of the Road",
+                "SSP 5-8.5 -- Burn it All",
+            ]:
+                if scenario_i in scenario_ssp_options:  # Reorder list
+                    scenario_ssp_options.remove(scenario_i)  # Remove item
+                    scenario_ssp_options.append(scenario_i)  # Add to back of list
+            self.param["scenario_ssp"].objects = scenario_ssp_options
+            self.scenario_ssp = [
+                x for x in self.scenario_ssp if x in scenario_ssp_options
+            ]
 
-        # check if input historical scenarios match new available scenarios
-        # if no reanalysis scenario then return False
-        def _check_inputs(a, b):
-            chk = False
-            if len(b) < 2:
+            historical_scenarios = ["historical", "reanalysis"]
+            scenario_historical_options = [
+                scenario_to_experiment_id(scen, reverse=True)
+                for scen in self.scenario_options
+                if scen in historical_scenarios
+            ]
+            self.param["scenario_historical"].objects = scenario_historical_options
+
+            # check if input historical scenarios match new available scenarios
+            # if no reanalysis scenario then return False
+            def _check_inputs(a, b):
+                chk = False
+                if len(b) < 2:
+                    return chk
+                for i in a:
+                    if i in a:
+                        chk = True
                 return chk
-            for i in a:
-                if i in a:
-                    chk = True
-            return chk
-
-        # check if new selection has the historical scenario options and if not select the first new option
-        if _check_inputs(_scenario_historical, scenario_historical_options):
-            self.scenario_historical = _scenario_historical
+
+            # check if new selection has the historical scenario options and if not select the first new option
+            if _check_inputs(_scenario_historical, scenario_historical_options):
+                self.scenario_historical = _scenario_historical
+            else:
+                self.scenario_historical = [scenario_historical_options[0]]
+
         else:
```

Comment:
nit: This might not be needed

Reply:
Not sure what this line of code is (I didn't add it, it's unrelated to WL stuff). I can look into it and address in a different PR if necessary